src/libraries/Native/AnyOS/System.IO.Compression.Native/entrypoints.c

```diff
@@ -1,7 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#include <common/entrypoints.h>
+#include <minipal/entrypoints.h>
```

Comment:
Can `entrypoints.h` include `minipal/utils.h`? It looks like one always needs `minimap/utils.h` when you use `entrypoints.h`.